lib/ddtrace/ext/distributed.rb

```diff
@@ -20,8 +20,10 @@ module DistributedTracing
       PROPAGATION_STYLE_DATADOG = 'Datadog'.freeze
       PROPAGATION_STYLE_B3 = 'B3'.freeze
       PROPAGATION_STYLE_B3_SINGLE_HEADER = 'B3 single header'.freeze
-      PROPAGATION_INJECT_STYLE_ENV = 'DD_PROPAGATION_INJECT_STYLE'.freeze
-      PROPAGATION_EXTRACT_STYLE_ENV = 'DD_PROPAGATION_EXTRACT_STYLE'.freeze
+      PROPAGATION_INJECT_STYLE_ENV = 'DD_PROPAGATION_STYLE_INJECT'.freeze
```

Comment:
Could you rename the variables to match the word order swap?

Reply:
makes sense will do